<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget</class>
 <widget class="QDockWidget" name="PropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>831</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>192</width>
    <height>240</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Edge</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEdgeType">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lanes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEdgeLanes">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>One Way</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEdgeOneWay">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEdgeLink">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Link</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Roundabout</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEdgeRoundabout">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Vertex</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVertexDesc">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVertexPos">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Neighbors</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditVertexNeighbors">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Example</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditKernel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parent</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Uncles</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVertexParent">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditVertexUncles">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>On boundary?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVertexOnBoundary">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
